packages/ruleset/test/rules/response-status-codes.test.js

```diff
@@ -230,16 +247,14 @@ describe(`Spectral rule: ${ruleId}`, () => {
       };
 
       const results = await testRule(ruleId, rule, testDocument);
-      expect(results).toHaveLength(1);
+      expect(results).toHaveLength(2);
 
       expect(results[0].code).toBe(ruleId);
-      expect(results[0].message).toBe(
+      expect(results[1].message).toBe(
         'Operation responses should use status code 303 or 307 instead of 302'
       );
       expect(results[0].severity).toBe(expectedSeverity);
-      expect(results[0].path.join('.')).toBe(
-        'paths./v1/drinks.post.responses.302'
-      );
```

Comment:
This error message has moved to the 2nd entry in the `results` list. It would good to keep checking it.